src/CMake/FindADIOS2.cmake

```diff
@@ -34,7 +34,8 @@ if(NOT WIN32)
         adios2_perfstubs adios2_cxx11 adios2_core adios2_enet)
 else()
     SET_UP_THIRD_PARTY(ADIOS2 LIBS
-        adios2_c adios2_cxx11 adios2_core adios2_perfstubs)
+        adios2_c adios2_cxx11 adios2_core adios2_atl
+        adios2_dill adios2_ffs )
```

Comment:
I thought these libs were not built with the latest adios2?

Reply:
They build on Windows. Unless I missed a flag that turns them off ? I will double check my Cmake build flags I used against bv_adios2.sh.

Reply:
Just checked a build I did on CZ, and those libs were built for adios2 there as well.
Seems there is a run-time dependency for VisIt on Windows, but possibly not on Linux ?

Reply:
Hmm, not sure why I didn't include them. I see them in my build on the RZ as well.